tests/utils/test_utils.py

```diff
@@ -13,8 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-from release_notes_generator.utils.utils import get_change_url, detect_row_format_invalid_keywords
+from release_notes_generator.action_inputs import ActionInputs
```

Comment:
Not used.

Reply:
Import removed in commit 3a77a9ec12c250ead4ad816666a634511d384a2e.